.github/workflows/doc_check.yml

```diff
@@ -0,0 +1,42 @@
+name: Documentation Check for Magick++
+
+on:
+  push:

```

Comment:
Please also add `paths:` and include this file so it will run every time we make a change to it. And do the same for `pull_request:` and include this file.

You can find more information about this in the docs or the `windows-release.yml` file.